src/top/state.rs

```diff
@@ -47,30 +51,24 @@ pub enum ConnectionStatus {
     // reconnect attempts
     Disconnected(u64),
     // Connection is working
-    Connected,
+    Connected(DateTime<Local>),
```

Comment:
This boils down to showing the time the connection was established, for if users want to take the total events/bytes values and divide them by how long `top` has been watching, etc.